integration/duckdb/CMakeLists.txt

```diff
@@ -66,17 +65,26 @@ if(LANCE_BUILD_PYTORCH)
       set(torch_SOURCE_DIR ${CMAKE_BINARY_DIR}/thirdparty/torch/)
     endif()
   else()
-    FetchContent_Declare(
-      Torch
-      URL https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-1.12.1%2Bcpu.zip
-    )
+    if(LANCE_BUILD_CUDA)
+      FetchContent_Declare(
+        Torch
+        URL https://download.pytorch.org/libtorch/cu117/libtorch-cxx11-abi-shared-with-deps-1.13.0%2Bcu117.zip
```

Comment:
is there a place to pin pytorch version? Or mention it in README or somewhere?

Btw, do we want to support more than 1 cuda version?

Reply:
we can pin pytorch in lance, but we can't control if they pip install something different. Since 1.13 just got released, we should be ok for a little while. We can also check the version when doing `import lance` and print out a warning or something?

Reply:
ok actually there's no good place to pin it. pytorch is not currently a dependency. i'll add a thing in README.